packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -227,28 +227,70 @@ function create_routes_and_nodes(cwd, config, fallback) {
 					config.kit.moduleExtensions
 				);
 
+				/**
+				 * @param {string} type
+				 * @param {string} existingFile
```

Comment:
we should rename this variable to follow the convention since it's internal
```suggestion
				 * @param {string} existing_file
```

Reply:
oops, fixed